src/plugins/intel_cpu/src/utils/general_utils.h

```diff
@@ -53,6 +59,15 @@ constexpr bool implication(bool cause, bool cond) {
     return !cause || !!cond;
 }
 
+template <typename T, typename... Ts>
+constexpr bool one_of_v = (std::is_same_v<T, Ts> || ...);
+
+template <typename T, typename... Ts>
+constexpr bool none_of_v = !one_of_v<T, Ts...>;
```

Comment:
This value cane be miss that is same as usage of `none_of` function as suffix suggest it is alias for it

Use std trait for this case

Reply:
Yes, I know the semantic is different but name so close. Maybe just pick different name  but is optional suggestion only.